footbot/main.py

```diff
@@ -262,13 +262,23 @@ def update_predictions_route():
     return "predictions updated"
 
 
-@app.route("/optimise_team/<entry>")
-def optimise_team_route(entry):
+@app.route("/optimise_team/<entry>", methods=["GET", "POST"])
+def optimise_team_route(entry, optimise_entry=team_selector.optimise_entry):
```

Comment:
Looks like we're taking to `optimise_entry` as a function so we can provide a mocked version in the tests?

Reply:
Is it necessary to specify `optimise_entry=team_selector.optimise_entry` rather than `optimise_entry=optimise_entry`, given that we `from footbot.optimiser import team_selector` above? Or is this just for readability? Asking for a friend...

Reply:
yes that's what we're doing- I left it as `team_selector.optimise_entry` as that's what it was using it before, but yes can just be `team_selector`